### PR TITLE
feat(cli,ironfish): Add flag to override name in round3

### PR DESCRIPTION
**Pull request by rohanjadvani**
## Summary

Allow an override of the secret name for the account name during round 3

## Testing Plan

Updated unit test

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and label it with `breaking-change-rpc` or `breaking-change-sdk`.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/rpc/routes/wallet/multisig/dkg/round3.ts [hunk 1]
```diff
@@ -28,6 +29,7 @@ export const DkgRound3RequestSchema: yup.ObjectSchema<DkgRound3Request> = yup
     round2SecretPackage: yup.string().defined(),
     round1PublicPackages: yup.array().of(yup.string().defined()).defined(),
     round2PublicPackages: yup.array().of(yup.string().defined()).defined(),
+    name: yup.string().optional(),
```
Comment (andiflabs) [inline ironfish/src/rpc/routes/wallet/multisig/dkg/round3.ts:32]: Nit: can we call this `accountName`?

File: ironfish-cli/src/commands/wallet/multisig/dkg/round3.ts [hunk 2]
```diff
@@ -17,6 +17,10 @@ export class DkgRound3Command extends IronfishCommand {
       description: 'The name of the secret to use for decryption during DKG',
       required: true,
     }),
+    name: Flags.string({
```
Comment (andiflabs) [inline ironfish-cli/src/commands/wallet/multisig/dkg/round3.ts:20]: Would `accountName` be a better name for the flag?

**Review by andiflabs — APPROVED** [review]

File: ironfish-cli/src/commands/wallet/multisig/dkg/round3.ts (hunk 2 above)
Comment (rohanjadvani) [reply ironfish-cli/src/commands/wallet/multisig/dkg/round3.ts]: Yeah I'm ok with that. Updated

File: ironfish/src/rpc/routes/wallet/multisig/dkg/round3.ts (hunk 1 above)
Comment (rohanjadvani) [reply ironfish/src/rpc/routes/wallet/multisig/dkg/round3.ts]: Yeah I'm ok with that. Updated

**Pull request merged**